index.js

```diff
@@ -1,12 +1,11 @@
-const fs = require('fs-extra');
+const fs = require('fs/promises');
```

Comment:
New in Node 14, but Node 12 is no longer supported, so I guess we can get away with it.